evm_arithmetization/src/prover.rs

```diff
@@ -66,6 +87,40 @@ impl GenerationSegmentData {
     }
 }
 
+pub fn zkevm_fast_config() -> StarkConfig {
+    let cap_height = 4;
+    let mut strategy = Vec::new();
+    for window in Table::all_degree_logs().windows(2) {
+        if window[0] != window[1] {
+            strategy.push(window[0] - window[1]);
+        }
+    }
```

Comment:
How do you intend to have it work with dynamic batching? The config must be known for preprocessing

Reply:
Hmm, good point. I guess the easiest way would be to add a `BatchedStark` strategy to compute this dynamically.